gobblin-utility/src/main/java/gobblin/util/limiter/broker/SharedLimiterFactory.java

```diff
@@ -60,11 +67,36 @@ public String getName() {
   }
 
   @Override
-  public ResourceInstance<Limiter> createResource(SharedResourcesBroker broker, ScopedConfigView<?, SharedLimiterKey> configView)
+  public SharedResourceFactoryResponse<Limiter>
+    createResource(SharedResourcesBroker broker, ScopedConfigView<?, SharedLimiterKey> configView)
       throws NotConfiguredException{
+
     Config config = configView.getConfig();
+    SharedLimiterKey.GlobalLimiterPolicy globalLimiterPolicy = configView.getKey().getGlobalLimiterPolicy();
+
+    if (config.hasPath(FAIL_IF_NO_GLOBAL_LIMITER_KEY) && config.getBoolean(FAIL_IF_NO_GLOBAL_LIMITER_KEY) &&
```

Comment:
So FAIL_IF_NO_GLOBAL_LIMITER_KEY has no effect when limiter policy is USE_LOCAL?